.mergify.yml

```diff
@@ -1,13 +1,12 @@
 pull_request_rules:
-  - name: Automatic merge
+  - name: Automatic merge to master
     conditions:
-      - "-draft"
-      - "#approved-reviews-by>=1"
-      - "approved-reviews-by=@gestalt-core"
-      - "status-success~=^Tests"
-      - "#status-failure=0"
+      - '-draft'
+      - '#approved-reviews-by>=1'
+      - 'base=master'
     actions:
       merge:
         strict: true
         strict_method: rebase
         method: squash
+        commit_message: title+body
```

Comment:
@chrislloyd do you have an example of what this looks like? What was the value before?

Reply:
No example. I'm just going by the documentation: https://doc.mergify.io/actions.html#merge